framework/integration/client/spec.go

```diff
@@ -0,0 +1,27 @@
+// +build k8srequired
+
+package client
+
+import "github.com/giantswarm/operatorkit/framework"
+
+type Interface interface {
```

Comment:
This here was basically my idea: An interface which allows us to do and get all the stuff we need in our testcases without the need to worry about different types. `configmap` and `nodeconfig` implement this interface and can then be used in the testcases.